composer.json

```diff
@@ -20,23 +20,22 @@
     "sabberworm/php-css-parser": "dev-master#cc791ad"
   },
   "require-dev": {
-    "automattic/vipwpcs": "^2.2",
+    "automattic/vipwpcs": "^3.0",
     "civicrm/composer-downloads-plugin": "^3.0",
-    "dealerdirect/phpcodesniffer-composer-installer": "0.7.2",
```

Comment:
No longer necessary?

Reply:
WPCS handles it well as they need to keep few standards in installation path.